src/styles/blocks/calendar.scss

```
@@ -0,0 +1,59 @@
+@import "../utils/variables";
+
+.calendar {
+  /* Display Flex & Box Model */
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: #{$day-gap};
+  width: #{$calendar-width};
+  height: auto;
+  padding: #{$calendar-padding};
+  padding-top: #{$calendar-padding-top};
+
+  /* for testing bg */
+  // background-color: beige;
+
+  &__day {
+    /* Display& Box Model */
+    box-sizing: border-box;
+    width: #{$day-size};
+    height: #{$day-size};
+    border: 1px solid black;
+    background-color: #{$bg-color-day};
+
+    /* Font */
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: #{$font-size-day};
+
+    /* Animation */
+    transition: all 0.5s;
+
+    &:hover {
+      transform: translate(
+        #{$animation-day-transorm-w},
+        #{$animation-day-transorm-h}
+      );
+      background-color: pink;
+      cursor: pointer;
+    }
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i}):before {
+        content: "#{$i}";
+      }
+    }
+
+    @each $name-day, $day in $start-day {
+      .calendar--start-day--#{$name-day} > &:first-child {
+        margin-left: calc(#{$day-size} * (#{$day} - 1) + (#{$day}px - 1px));
+      }
+    }
+
+    @each $days in $month-length {
+      .calendar--days--#{$days} > &:nth-child(n + #{$days + 1}) {
+        display: none;
+      }
```

```suggestion
    @each $name-day, $day in $start-day {
      &--start-day--#{$name-day} > &:first-child {
        margin-left: calc(#{$day-size} * (#{$day} - 1) + (#{$day}px - 1px));
      }
    }

    @each $days in $month-length {
      &--days--#{$days} > &:nth-child(n + #{$days + 1}) {
        display: none;
      }
```
we're in the calendar block so we can probably use it like that 😄

---

It's not working, bc we are in the <.calendar__day {}> 
But I moved it and changed it. :)
And now it looks like that:
 `  @each $name-day, $day in $start-day {
    &--start-day--#{$name-day} > &__day:first-child {
      margin-left: calc(#{$day-size} * (#{$day} - 1) + (#{$day}px - 1px));
    }
  }

  @each $days in $month-length {
    &--days--#{$days} > &__day:nth-child(n + #{$days + 1}) {
      display: none;
    }
  }`